### ISSUE TITLE
Is it possible to somehow start the h2 inmemory db by integrating Serenity with Spring?

### ISSUE THREAD

**Issue by beetlezhuk** [serenity-bdd/serenity-core#1510]
For our project, we need to connect tests to some db. Ideally, would be to use h2 in-memory db. However, other ideas are welcome.

**Comment by wakaleo** [comment]
You can integrate Serenity with Spring quite easily - see https://github.com/serenity-bdd/serenity-documentation/blob/master/src/asciidoc/spring.adoc

**Comment by beetlezhuk** [comment]
Yes, agree. But I cannot start h2 db for the test. Is there any way to do that?

**Comment by wakaleo** [comment]
I don't know offhand, you would need to check the h2 documentation.
